db/migrate/20160922090347_remove_field_region_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveFieldRegionFromUsers < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :users, :region, :integer
```

Comment:
- Please explicitly drop the index on `[:region, :userid]` first
- Please add a non-unique index on `[:userid]`.